layers-samples/proj-multiview-occlusion.html

```diff
@@ -0,0 +1,303 @@
+IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+-->
+<html>
+
+<head>
+  <meta charset='utf-8'>
+  <meta name='viewport' content='width=device-width, initial-scale=1, user-scalable=no'>
+  <meta name='mobile-web-app-capable' content='yes'>
+  <meta name='apple-mobile-web-app-capable' content='yes'>
+  <link rel='icon' type='image/png' sizes='32x32' href='../favicon-32x32.png'>
+  <link rel='icon' type='image/png' sizes='96x96' href='../favicon-96x96.png'>
+  <link rel='stylesheet' href='../css/common.css'>
+
+  <title>Projection Layer with Multiview</title>
+</head>
+
+<body>
+  <header>
+    <details open>
+      <summary>Projection Layer with Multiview</summary>
+      <p>
+        This sample demonstrates use of WebXR depth sensing to do simple occlusion.
+        <a class="back" href="./">Back</a>
+      </p>
+      <p id="mv-status"></p>
+      <input type="checkbox" id="do_stencil_buffer">stencil buffer</input>
+      <input type="checkbox" id="do_antialias" checked>antialias</input>
+    </details>
+  </header>
+  <script type="module">
+    import { WebXRButton } from '../js/util/webxr-button.js';
+    import { Scene, WebXRView } from '../js/render/scenes/scene.js';
+    import { Renderer, createWebGLContext } from '../js/render/core/renderer.js';
+    import { CubeSeaNode } from '../js/render/nodes/cube-sea-occlusion.js';
+    import { InlineViewerHelper } from '../js/util/inline-viewer-helper.js';
+    import { QueryArgs } from '../js/util/query-args.js';
+
+    // If requested, use the polyfill to provide support for mobile devices
+    // and devices which only support WebVR.
+    import WebXRPolyfill from '../js/third-party/webxr-polyfill/build/webxr-polyfill.module.js';
+    if (QueryArgs.getBool('usePolyfill', true)) {
+      let polyfill = new WebXRPolyfill();
+    }
+
+    // XR globals.
+    let do_stencil_buffer = document.getElementById('do_stencil_buffer');
+    let do_antialias = document.getElementById('do_antialias');
+    let xrButton = null;
+    let xrImmersiveRefSpace = null;
+    let inlineViewerHelper = null;
+    let xrGLFactory = null;
+    let xrFramebuffer = null;
+
+    // WebGL scene globals.
+    let gl = null;
+    let renderer = null;
+    let scene = new Scene();
+    let is_multisampled_supported = false;
+    let samples = 1;
+    let mv_ext = null;
+    let depthStencilTex = null;
+
+    scene.addNode(new CubeSeaNode({ imageUrl: '../media/textures/cube-sea.png' }));
+    scene.enableStats(false);
+
+    function initXR() {
+      xrButton = new WebXRButton({
+        onRequestSession: onRequestSession,
+        onEndSession: onEndSession
+      });
+      document.querySelector('header').appendChild(xrButton.domElement);
+
+      if (navigator.xr) {
+        navigator.xr.isSessionSupported('immersive-ar').then((supported) => {
+          let mvCompat = testMultiViewCompatibility();
```

Comment:
Give that this sample is focused on occlusion, not multiview, can we switch this to test for and inform the user about the availability of the depth texture feature?

Reply:
done

Reply:
I can't test for availability of the depth texture feature until requestSession time.
I *could* make it work without multiview (like I did for three.js) if needed.

Reply:
Ah, okay. If multiview is required then it's fine to leave this here, but I would suggest changing the user-facing message in a follow-up PR to have it say something along the lines of "Multiview is required but not supported", since at the moment it's a little unclear _why_ the page is bringing up Multiview at all.